cloud-controlplane/x-topics/quickstart.md

```diff
@@ -24,17 +24,27 @@ If you successfully retrieve an access token, it is valid for one hour. You can
 
 ### BYOC or Dedicated
 
-1. In the page header, click **API Explorer**.
+1. Open **API Explorer**.
```

Comment:
```suggestion
1. Open **API Explorer** in the menu bar. 
```
or, In the menu bar, select API Explorer?

Reply:
Updated